packages/components/src/ObjectViewer/List/List.component.js

```diff
@@ -5,12 +5,14 @@ import JSONLike from '../JSONLike';
 import theme from './List.scss';
 
 function List({ data, ...props }) {
-	if (!Array.isArray(data)) {
+	if (!data || (!Array.isArray(data) && !Array.isArray(data.datas))) {
```

Comment:
remove parenthesis (!Array.isArray(data) && !Array.isArray(data.datas). AND is priority.